artistools/misc.py

```diff
@@ -258,6 +258,27 @@ def get_vpkt_config(modelpath: Path | str) -> dict[str, t.Any]:
             int(x) for x in vpkt_txt.readline().split()
         )
 
+        # read the next line
+        vpackets_total_active_spectral_region = vpkt_txt.readline().split()
+        vpkt_config["vpackets total active spectral region"] = vpackets_total_active_spectral_region
+
+        optically_thick_cells = vpkt_txt.readline().split()
+        vpkt_config["optically thick cells"] = optically_thick_cells
```

Comment:
I guess you would have been looking at the artis source code to determine the meaning of items in the vpkt.txt file. Artis calls the items on this line override_thickcell_tau and cell_is_optically_thick_vpkt, which I find more description than "optically thick cells".

Please follow the existing convention and avoid line noise, e.g.,
```python
vpkt_config["override_thickcell_tau"], vpkt["cell_is_optically_thick_vpkt"] = vpkt_txt.readline().split()
```

Similarly for the following lines, consider directly re-using the artis variable names for consistency, or coming up with a more descriptive name while avoiding spaces in the dictionary keys to match the existing code.